### PR TITLE
Fix max length issue on text inputs by always forcing the length of v…

### PR DESCRIPTION
**Pull request by mallenscdhhs**
…alue to be no more than the supplied max property(if supplied).

### CODE REVIEW

**Comment by mallenscdhhs** [comment]
...also, ES6-ify the input-spec

File: src/Input.jsx [hunk 1]
```diff
@@ -99,14 +99,23 @@ export default React.createClass({
...
+    } else {
+      changeEvent.target.value = this.forceMaxLength(event.target.value);
     }
     this.onChange(changeEvent);
   },
 
+  forceMaxLength(value) {
+    let max = Number(this.props.max);
+    return max? value.slice(0, max) : value;
+  },
```
Comment (traviscooperscdhhs) [inline src/Input.jsx]: The 'max' prop can also refer to numerical values in the Input, not just 'length'.  '/CR/VR/VR_NumBetweenMinMaxParam' is a rule that uses that.  I think we may want to use 'maxLength' and use it to stop you from entering in more chars. And let the different rules use the 'min' 'max' props to customize the validation rules.

Comment (mallenscdhhs) [inline src/Input.jsx]: good point, I will fix

**Comment by traviscooperscdhhs** [comment]
:+1:

**Pull request closed without merging**